scripts/copy_druid_docs.py

```diff
@@ -57,13 +59,14 @@ def check_source(source_directory):
 
 def replace_text_in_file(destination_directory, druid_version):
     """
-    Find/replace {{DRUIDVERSION}} with the actual version
+    Find/replace "{{DRUIDVERSION}}", "\{DRUIDVERSION}", "\{{DRUIDVERSION}}", "\{DRUIDVERSION}}"
```

Comment:
Should everything within the quotes be captured? Is it capturing the `/`?

Not sure if testing correctly but I see only the first one fully captured